packages/core/saved-objects/core-saved-objects-api-server/src/apis/resolve.ts

```diff
@@ -17,6 +17,7 @@ import type { SavedObject } from '../..';
 export interface SavedObjectsResolveOptions extends SavedObjectsBaseOptions {
   /** {@link SavedObjectsRawDocParseOptions.migrationVersionCompatibility} */
   migrationVersionCompatibility?: 'compatible' | 'raw';
+  downwardConversion?: 'allow' | 'forbid';
```

Comment:
NIT: extract / factorize the type definition to use the same between the `find`, `get` and `resolve` definitions.

Reply:
`find` options include many more than `get` and `resolve` and I'm on the fence about re-using a factorized type definition for it too.